addons/website_sale/controllers/backend.py

```diff
@@ -90,6 +90,7 @@ def fetch_dashboard_data(self, date_from, date_to):
                 ('order_line', '!=', 'False'),
                 ('partner_id', '!=', request.env.ref('base.public_partner').id),
                 ('date_order', '<=', fields.Datetime.to_string(datetime.now() - timedelta(hours=1))),
+                ('partner_id', '!=', request.env.ref('base.public_partner').id),
```

Comment:
In your wizard hereunder, you add `('team_id.team_type', '=', 'website')` in the domain